aiohttp/web_fileresponse.py

```diff
@@ -337,7 +334,7 @@ async def _prepare_open_file(
 
             # If a range request has been made, convert start, end slice
             # notation into file pointer offset and count
-            if start is not None or end is not None:
+            if start is not None:
```

Comment:
`end` is checked in the next if. It only makes sense to enter this block is `start is not None` since all the operations with `start` require it to be an `int`